pipelines/hacbs/hacbs-test.yaml

```diff
@@ -101,3 +101,14 @@
     workspaces:
     - name: workspace
       workspace: workspace
+- op: add
+  path: /spec/tasks/-
+  value:
+    name: snyk-code-test
```

Comment:
I think we should go with the convention to prefix the task name with `sast-`, so I'm proposing naming this task `sast-synk-check`

Reply:
Updated